docs/bidders/pubnative.md

```diff
@@ -0,0 +1,58 @@
+# Pubnative Bidder
+
+## Prerequisite
+Before adding PubNative as a new bidder, there are 3 prerequisites:
+- As a Publisher, you need to have Prebid Mobile SDK integrated.
+- You need a configured Prebid Server (either self-hosted or hosted by 3rd party).
+- You need to be integrated with Ad Server SDK (e.g. Mopub) or internal product which communicates with Prebid Mobile SDK.
```

Comment:
Are there any Pubnative help resources for people to consult in case they want to implement these?

Reply:
Added, txs.